Adding a hyperlink for two references in the README.md (#47132)
freeCodeCamp/freeCodeCamp: README.md

Each of these 50 projects in the freeCodeCamp.org curriculum has its own agile user stories and automated tests. These help you build up your project incrementally and ensure you've fulfilled all the user stories before you submit it.

> You can pull in these test suites through [freeCodeCamp's CDN](https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js). This means you can build these projects on websites like [CodePen](https://codepen.io) and [Replit](https://replit.com) - or even on your local computer's development environment.

Reviewer: ```suggestion
You can pull in these test suites through [freeCodeCamp's CDN](https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js). This means you can build these projects on your cloud development environment of your choice  - or on your local computer's development environment.
```
